guide/case-studies/blixt-wallet.md

```diff
@@ -0,0 +1,173 @@
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-wallet-usability-testing-data-overview-mobile@2x.png"
+   alt-text = "Chart depicting an overview of the data from the usability tests, showing user confusion about usernames, opening channels, the copywriting, and how to fund the wallet by copying a QR code."
+   width = 800
+   height = 751
+%}
+
+- Users new to lightning are missing a crucial step, the why. They often lack understanding on why they need to open a channel before being presented with a screen to open one. Once this is clear it becomes easy for a user to understand that this step is necessary before transacting.
+*For more information on channels, see the [Technology Primer]({{'/guide/getting-started/technology-primer/#what-is-a-payment-channel' | relative_url}}) and [Lightning Liquidity]({{'/guide/how-it-works/liquidity/' | relative_url}}) section.*
+- Explaining to users how long it takes to open a channel sets a clear expectation around the process. This would be best done early on during the onboarding journey.
+
+Opening channels is a process unique to lightning wallets and an essential first step during the onboarding process. For users brand-new to lightning, opening a channel in itself is something that is new and unique to them. It is then helpful to explain to users why a channel should be opened before presenting them with the channel opening screen An example of the text could read:
+
+### Explaining the concept of opening a channel to a user that has never used a lightning wallet
+
+“First things first, to use your lightning wallet, you’ll need to set up a channel. Think of a channel as your electricity connection to the lightning network. By opening a channel you are able to make transactions.
+
+To open a channel first we will need to add some bitcoin to it. Once that is done you will be ready to make your first transaction.”
+
+{% include picture.html
+   image = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test.png"
+   retina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test@2x.png"
+   mobile = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-mobile.jpngpg"
+   mobileRetina = "/assets/images/guide/case-studies/blixt-wallet/blixt-usability-test-mobile@2x.png"
+   alt-text = "Diagram showing comments from users and which section of the UI they were commenting on"
+   width = 800
+   height = 751
+%}
+
+## Here’s what we learned about educating users
+
+- Providing users with educational screenstates is a good idea. It’s always nice to keep the text as short as possible and the explanation easy. Through the usability testing, we learned that we could further simplify the text in this educational part.
+- We also learned that when providing educational screenstates we should allow users the option of going back, so that they can read the text from the previous screen.
```

Comment:
```suggestion
- We also learned that when providing educational screen states, we should allow users the option of going back, so that they can read the text from the previous screen.
```